rest-api-model-server/build.gradle.kts

```diff
@@ -24,9 +26,12 @@ dependencies {
     implementation("io.quarkus:quarkus-arc")
     implementation("org.modelix:model-client:$modelix_version")
     implementation("org.modelix.mps.api-gen:runtime:$api_gen_version")
-    implementation(project(":mps:solutions:University.Schedule.api"))
+
     testImplementation("io.quarkus:quarkus-junit5")
     testImplementation("io.rest-assured:rest-assured")
+
+    implementation(project(":mps:metamodel"))
+    implementation("org.modelix:metamodel-runtime:$modelix_metamodel_version")
```

Comment:
Same as above: make this a dependency of the metamodel subproject already.

Reply:
Btw, you need to use `api` then instead of `implementation` to expose the dependency to client modules.

Reply:
we can do without line 34

regarding `api`: we do not want to expose here, so `implementation` is fine?

Reply:
Ah yes, my idea was to use `api` in the `metamodel` module, because the generated metamodel is not usable without the runtime. Therefore, it should expose this dependency. Here, the explicit dependency on the metamodel-runtime can then be dropped